staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -96,9 +96,10 @@ func handleRootHealthz(checks ...HealthzChecker) http.HandlerFunc {
 		failed := false
 		var verboseOut bytes.Buffer
 		for _, check := range checks {
-			err := check.Check(r)
-			if err != nil {
-				fmt.Fprintf(&verboseOut, "[-]%v failed: %v\n", check.Name(), err)
+			if check.Check(r) != nil {
```

Comment:
This probably breaks backwards compatibility on health checks, and people may already be using these checks in monitoring.  Effectively an API change - you may have to print something like "reason withheld" in the same format as before at a minimum.